docs/common/DocsPageTemplate/Header.vue

```diff
@@ -6,6 +6,9 @@
   >
     <div>
       <h1 class="header-text">
+        <span @click="toggleSideNav">
+        <KIcon icon="menu"  />
```

Comment:
[`KIconButton`](https://deploy-preview-862--kolibri-design-system.netlify.app/kiconbutton/) is a component that'd be appropriate to use for hamburger and close interactive icons

Reply:
Sure